status/status.md

```diff
@@ -30,7 +30,7 @@ Status | Config | Repo
 [![Build Status](https://travis-ci.org/JStumpp/awesome-android.svg)](https://travis-ci.org/JStumpp/awesome-android) | [`config`](https://github.com/JStumpp/awesome-android/blob/master/.travis.yml) | [JStumpp/awesome-android](https://github.com/JStumpp/awesome-android) 
 [![Build Status](https://travis-ci.org/humiaozuzu/awesome-flask.svg)](https://travis-ci.org/humiaozuzu/awesome-flask) | [`config`](https://github.com/humiaozuzu/awesome-flask/blob/master/.travis.yml) | [humiaozuzu/awesome-flask](https://github.com/humiaozuzu/awesome-flask) 
 [![Build Status](https://travis-ci.org/j-delaney/easy-application.svg)](https://travis-ci.org/j-delaney/easy-application) | [`config`](https://github.com/j-delaney/easy-application/blob/master/.travis.yml) | [j-delaney/easy-application](https://github.com/j-delaney/easy-application) 
-[![Build Status](https://travis-ci.org/AngularClass/awesome-angular2.svg)](https://travis-ci.org/AngularClass/awesome-angular2) | [`config`](https://github.com/AngularClass/awesome-angular2/blob/master/.travis.yml) | [AngularClass/awesome-angular2](https://github.com/AngularClass/awesome-angular2) 
+[![Build Status](https://travis-ci.org/AngularClass/awesome-angular.svg)](https://travis-ci.org/AngularClass/awesome-angular) | [`config`](https://github.com/AngularClass/awesome-angular/blob/gh-pages/.travis.yml) | [AngularClass/awesome-angular2](https://github.com/AngularClass/awesome-angular) 
```

Comment:
can you change one more thing, [AngularClass/awesome-angular2] to [AngularClass/awesome-angular]

thanks!